chemie/lockers/views.py

```diff
@@ -84,7 +94,15 @@ def register_locker(request, number):
             # Create a new ownership for the user
             new_ownership = Ownership(locker=locker, user=user)
             if new_ownership.reached_limit():
-                raise Http404
+                messages.add_message(
+                    request,
+                    messages.ERROR,
+                    "Du har allerede to registrerte bokskap. "
+                    "For å kunne reservere et nytt må du fjerne ett av de gamle.",
```

Comment:
Change "ett av de gamle" to "et av de gamle"? He may also remove both, but this is me being picky.